src/js/index.js

```diff
@@ -14,9 +14,9 @@
       const target = this.hash;
       const $target = $(target); // use hash as a selector to find the section by id
       $('html, body').stop().animate({
-        'scrollTop': $target.offset().top
-      }, 700, 'swing', function () {
-        window.location.hash = target;
+        'scrollTop': $('.navbar').css('position') === 'fixed' ? $target.offset().top - $('.navbar').height() : $target.offset().top // scroll to section location minus the height of navbar
+    }, 700, 'swing', function () {
+        window.location.hash = $('.navbar').css('position') === 'fixed' ? $target.offset().top - $('.navbar').height() : target; // end at section location minus the height of navbar
```

Comment:
I don't think this change is needed. It should set the hash to the section id so people can easily copy the deeplink from the address bar!

![image](https://user-images.githubusercontent.com/22859824/141187548-a6b688e8-5c7e-4e50-bea8-a75c1aaf7f33.png)

Reply:
so let the scrolling be broken or is there another way to fix it? I know we're looking at adding and splitting some of the content to different pages on the website soon

Reply:
I think your change on 17 fixes scrolling (so that it matches with the top of the section - the header height). This line should stay `target` so the hash is set correctly

or was it broken beyond that?

Reply:
With just the change on 17, it scrolls to the correct point on the page but then quickly snaps to cover the top of the section again. If there isn't a way to fix it without messing with the hash, I can remove the 'position: fixed' from .navbar for now.

Reply:
ohh super interesting, it must be doing the default browser behavior of `onhashchange` and snap the page to the given hash

I think if we wanted to keep the fixed header we'd need to assign a handler for this: https://developer.mozilla.org/en-US/docs/Web/API/WindowEventHandlers/onhashchange

In this handler, we'll want to prevent the default behavior and instead snap to the top position that has the offset accounted for

Reply:
Here's another option: https://stackoverflow.com/questions/10732690/offsetting-an-html-anchor-to-adjust-for-fixed-header/13067009